### ISSUE TITLE
Logout URL is not correct and leads to a 404 not found

### ISSUE THREAD

**Issue by nohanna** [ory/kratos#2210]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

I followed the [quickstart tutorial](https://www.ory.sh/login-spa-react-nextjs-authentication-example-api-open-source/) with Next.js to try with my test application.

Everything seems to work fine, my identities are created and I can sign in with my users. The issue is that when I switch from:
```
export default createApiHandler({
  fallbackToPlayground: true,
});
```
to
```
export default createApiHandler({
  apiBaseUrlOverride: 'https://my.api.from.ory.console',
  fallbackToPlayground: false,
});
```
my logout is not working anymore (and the URL seems correct) and leads to a 404 page.

Note that, instead of deploying on GitHub as suggested, I just wanted to try in local, so I used the `ory proxy` command to work with localhost:
```
export ORY_SDK_URL=https://my.api.from.ory.console
ory proxy http://localhost:3000
```

I'm not sure if I missed something, but since everything else is working, I guess it's a bug.

### Reproducing the bug

Create a new project and follow the tutorial (but instead of deploying your app on GitHub, use `ory proxy` in local).
Use your configuration as follow:
```
export default createApiHandler({
  apiBaseUrlOverride: 'https://my.api.from.ory.console',
  fallbackToPlayground: false,
});
```

Then register a user, sign in and then try to logout. It will lead to a 404 page.

### Version

0.8.2-alpha.1

### On which operating system are you observing this issue?

macOS

**Comment by aeneasr** [comment]
Hey there, thank you for the report! Could you share the 404 page URL that you end up with? And maybe record a small video that shows the flow which breaks for you? Thank you! :)

Also, which Ory CLI version do you have? `ory version` can give that info :)

**Comment by nohanna** [comment]
Hi @aeneasr! Sure, I ended up there: http://localhost:4000/api/.ory/ui/error?id=0da6f6d4-4271-482b-8a78-93da8250a23a
I have Ory CLI version 0.1.22.

Here is a small [record of my screen](https://user-images.githubusercontent.com/11637244/153612069-d2332836-ce52-4a6c-972b-d2880a63a0f7.mov).

Hope this helps! :)

**Comment by aeneasr** [comment]
Thank you for the tutorial! You are probably referring to this tutorial correct?

https://www.ory.sh/login-spa-react-nextjs-authentication-example-api-open-source/

I will try and reproduce it today.

ps: for NextJS you don't need the Ory Proxy :)

**Comment by nohanna** [comment]
Exactly, it is this tutorial.
Oh ok, thanks for your help! :)

**Comment by aeneasr** [comment]
Hey! I just tried this and was not able to reproduce the issue:

```
git clone https://github.com/ory/kratos-nextjs-react-example.git
cd kratos-nextjs-react-example
npm i
export ORY_SDK_URL=https://keen-hodgkin-1f3ozs1mpm.projects.oryapis.com <- this is a test project i created
npm run dev
```

alternatively you can also do:

```
// And create the Ory Cloud API "bridge".
export default createApiHandler({
  apiBaseUrlOverride: 'https://keen-hodgkin-1f3ozs1mpm.projects.oryapis.com',
  fallbackToPlayground: true
})
```

And then I was able to complete the logout flows etc :)

Let me know if this worked for you, and if it did, what we need to improve so that the docs are less confusing about this!

**Comment by nohanna** [comment]
Hi! I actually removed the Ory Proxy and only used as you suggested:
```
export default createApiHandler({
  apiBaseUrlOverride: 'https://my.api.from.ory.console',
  fallbackToPlayground: true
})
```
And it worked! I guess there is a conflict when I used both at the same time.
Your docs are really great, the thing is when I tried it the first time, it didn't work and it was still calling the playground API, that's why I thought I had to use the Ory Proxy! I think I had some issues with my code, but it works now :)

Thanks for your time!
